jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
```

Comment:
Add `nbtime==3.1.0` since it's checked by [ods-ci framework](https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Tests/500__jupyterhub/plugin-verification.robot#L16).